.aspect/cli/plugins.yaml

```diff
@@ -0,0 +1,13 @@
+# Plugins fail on CI: https://github.com/aspect-build/rules_js/actions/runs/3451833892/jobs/5761331944
```

Comment:
Upgrading caused this? So it's a cli issue?

Reply:
The issue exists on both 4.2.1 & 4.2.2 of the CLI. Just adding the plugins breaks.